helpers/database.ts

```diff
@@ -0,0 +1,28 @@
+import { connect, connection, model, Schema } from 'mongoose';
```

Comment:
Should we consider adding subfolders where appropriate? For example, I would have preferred placing this under a folder like `test/helpers/`, where `test` holds all `.spec.ts` files. Such a refactor may not be appropriate for this PR, but just figured I would propose the idea now, since it is relevant to this and has been on my mind for a while.

Reply:
@kezike - yes, I think it's totally fine to add subfolders in the test/ dir, for stuff like this.